checkov/terraform/checks/graph_checks/aws/S3BucketReplicationConfiguration.yaml

```diff
@@ -0,0 +1,91 @@
+metadata:
+  name: "Ensure that S3 bucket has cross-region replication enabled"
+  category: "GENERAL_SECURITY"
+  id: "CKV_AWS_144"
+definition:
+  or:
+    - cond_type: attribute
+      resource_types:
+        - aws_s3_bucket
+      attribute: replication_configuration.rules.*.status
```

Comment:
@gruebel why do u need the `*`?

Reply:
you can actually define several different rules. So, I'm actually not sure, what would be better to check, if one of them is `Enabled` or all of them. On the other hand, why should I create a rule, which is `Disabled` 🤷  so I decided to add the wildcard here 🙂

Reply:
@gruebel  first i see in tf v4 doc that the key is `rule`. and its an object, not a list

Reply:
correct, this one is the old way to do it, which used `rules` and in the new resource `aws_s3_bucket_replication_configuration` it is rule, which you can find at the bottom of the file. As the docs of the resource describe

> [rule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_replication_configuration#rule) - (Required) List of configuration blocks describing the rules managing the replication [documented below](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_replication_configuration#rule).

so you can add multiple `rule` blocks, if you want 🙂

Reply:
and here for the old way to do it directly in the `aws_s3_bucket` resource

> [rules](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#rules) - (Required) Specifies the rules managing the replication ([documented below](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#rules)).

pretty confusing 😄